pkg/controllers/nodeclaim/lifecycle/liveness.go

```diff
@@ -51,9 +51,14 @@ func (l *Liveness) Reconcile(ctx context.Context, nodeClaim *v1.NodeClaim) (reco
 	if registered == nil {
 		return reconcile.Result{Requeue: true}, nil
 	}
+	nodeRegistrationTTL := registrationTTL
```

Comment:
i am keeping the original registrationTTL because if users didn't apply the CRD first the default from the CRD won't be applied by apiserver. This makes sure we keep the 15 minute TTL regardless of is user updated the CRD

Reply:
Would it make sense to just have both then?

Reply:
I think it make sense on the nodepool side but not nodeclaim because spec of nodeclaim is immutable and the default would fail to apply (ran into this during testing)